lab/lab1/part3-data-transformation.html

```diff
@@ -71,6 +71,15 @@
 
     ===================== */
 
+    //Code support found via: http://adripofjavascript.com/blog/drips/filtering-arrays-with-array-filter.html
+    //Worked with Ian Schwarzenberg
```

Comment:
Thanks for calling this out! It's helpful. And we certainly encourage you to work together.